<?php try{}catch(Exception $b) {}
